packages/react-native-bpk-component-flat-list/README.md

```diff
@@ -87,6 +87,10 @@ Inherits all props from React Native's [FlatList](https://facebook.github.io/rea
 | selected           | bool                                  | false    | false         |
 | titleProps         | object                                | false    | {}            |
 
+#### `titleProps`
+
+`titleProps` is passed down to the `BpkText` used for the title. It accepts anything that React Native's [`Text` component](https://facebook.github.io/react-native/docs/text.html#props) does.
```

Comment:
Why don't we link to BpkText props instead of ReactNative?

Reply:
Because the RN one has an exhaustive list of all props and the BpkText one doesn't. So they'd end up on the RN one eventually anyway.

Reply:
Cool